### ISSUE TITLE
Exported animation keys are off by one

### ISSUE THREAD

**Issue by kungfooman** [KhronosGroup/glTF-Blender-Exporter#322]
Hi, I made a simple door model and a torus following the door. In Blender it looks as it is supposed to, but in three other GLTF viewers (threejs/babylon/win10 3d builder) it looks like the keys got a "off by one" error or something. Either three viewers got the same bug or the bug is in the blender exporter.

I made a video for better description: https://youtu.be/H1nRE2ipqQU

[![animation bug](https://img.youtube.com/vi/H1nRE2ipqQU/0.jpg)](https://www.youtube.com/watch?v=H1nRE2ipqQU)

I uploaded door.gltf, door.bin and door.blend here when anybody wants to use this as simple/small reference bug: http://killtube.org/downloads/gltf/door/ (asset is public domain, i just made it for debugging all the gltf animation problems)

**Comment by kungfooman** [comment]
I studied the Python exporter source somewhat in order to get a clue about the issue and I often stumbled over e.g.

```JS
key_offset = bpy.context.scene.frame_start / bpy.context.scene.render.fps
```

`bpy.context.scene.frame_start` was '1' the whole time, till I realized... that's probably the problem: and so it was. I set it to 0 and the first keyframes aren't cut anymore.

IMO this is very confusing though, why does a value in the timeline cuts off keyframes in separate NLA actions? I don't know if this behaviour is wanted.
